munge.R

```diff
@@ -130,8 +130,16 @@ pad <- pad %>%
 pad <- pad %>%
   replace_na(list(addrtype = 'OTHER', validlgcs = ''))
 
+<<<<<<< Updated upstream
```

Comment:
@chriswhong i'm confused why these merge conflict pieces are committed here? I think this will cause the script to break

Reply:
oops, will fix